src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -52,6 +52,13 @@ protected internal static bool TryParseLogLevel(string value, out LogLevel level
 			}
 		}
 
+		protected bool ParseVerifyServerCert(ConfigurationKeyValue kv)
+		{
+			if (kv == null || string.IsNullOrEmpty(kv.Value)) return true;
```

Comment:
I would add a named constant for the default value - https://github.com/elastic/apm-agent-dotnet/blob/master/src/Elastic.Apm/Config/ConfigConsts.cs#L13

Reply:
++